packages/flutter/test/material/chip_test.dart

```diff
@@ -1148,17 +1148,99 @@ void main() {
     expect(labelStyle.style.color, equals(Colors.black.withAlpha(0xde)));
   });
 
-  testWidgets('Chip semantics', (WidgetTester tester) async {
+  testWidgets('Chip semantics - default', (WidgetTester tester) async {
     final SemanticsTester semanticsTester = new SemanticsTester(tester);
 
     await tester.pumpWidget(new MaterialApp(
       home: const Material(
-        child: const Chip(
+        child: const RawChip(
```

Comment:
Why this switch to RawChip here and below?

Reply:
Since every other chip just passes values to RawChip it seems like the better place to test semantics